Task2/build.gradle

```diff
@@ -0,0 +1,38 @@
+plugins {
+    id 'java'
+}
+
+group 'com.epam.esm'
+version '1.0-SNAPSHOT'
+
+sourceCompatibility = 1.8
+
+ext {
+    junitVersion = '5.7.1'
+    springVersion = '5.3.5'
+}
+
+subprojects { project ->
+    project.apply plugin: 'java'
+
+    repositories {
+        mavenCentral()
+    }
+
+    dependencies {
+        implementation group: 'org.springframework', name: 'spring-webmvc', version: '5.3.5'
+
+        testImplementation group: 'org.junit.jupiter', name: 'junit-jupiter-api', version: "$junitVersion"
+        testRuntimeOnly("org.junit.jupiter:junit-jupiter-engine:$junitVersion")
+        testImplementation group: 'org.junit.jupiter', name: 'junit-jupiter-params', version: "$junitVersion"
+        testImplementation group: 'org.springframework', name: 'spring-test', version: "$springVersion"
+        testImplementation group: 'org.mockito', name: 'mockito-core', version: '3.9.0'
```

Comment:
@Pokemon3108 Please add versions to the `gradle.properties` as you did with other